README.md

```diff
@@ -22,6 +27,7 @@ usage: elicznik [-h] [--format {table,csv}] [--api {chart,csv}] username passwor
 positional arguments:
   username              tauron-dystrybucja.pl user name
   password              tauron-dystrybucja.pl password
+  site                  tauron-dystrybucja site. XXXXXXXXX_YYYYYYYY_ZZZZZZ format. 
```

Comment:
`start_date` and `end_date` have been the optional positional arguments so far.  Putting `site` before them will:
* break backward compatibility
* force users to specify a `site` if they only want to specify a start date

How about making it a switch option (e.g. `--site=...`).

Reply:
Swtiched to an optional `--site` as suggested. Now that I think of it it makes more sense than a third arg.